whipper/test/test_common_mbngs.py

```diff
@@ -114,3 +114,40 @@ def testMalaInCuba(self):
                           ';ec07a209-55ff-4084-bc41-9d4d1764e075'
                           ';f626b92e-07b1-4a19-ad13-c09d690db66c'
                           )
+
+    def testNorthernGateway(self):
+        # disc with artists tagged with [unknown] and an alias in MusicBrainz
+        # see https://github.com/JoeLametta/whipper/issues/155
```

Comment:
Make this a [docstring](http://www.pythonforbeginners.com/basics/python-docstrings/) instead of a comment. :)